### ISSUE TITLE
Blank red `error:` bar intermittently showing in DAG, localhost UI briefly became unavailable

### ISSUE THREAD

**Issue by aspain** [apache/airflow#23017]
### Apache Airflow version

2.2.5 (latest released)

### What happened

I was running a localhost instance of Astro Runtime 4.24, based on Airflow 2.2.5 and was testing two DAGs. 

During a DAG run, a blank red `error:` bar appeared briefly (it went away and came back several times). Screenshot of the error bar below along with the output in the network console at the time. 
![image](https://user-images.githubusercontent.com/15913202/163407951-aa53134f-2a17-484b-8c16-2d09b46b2d84.png)

At one point, when I refreshed the UI, it became entirely unavailable for about 1 minute:
![image](https://user-images.githubusercontent.com/15913202/163408470-cbac1f17-87c8-45b2-a492-5e647cd21161.png)
 

Eventually the page became available again, and the DAGs ran successfully.

### What you think should happen instead

There is no actual error displayed, just text `error:` with an empty red bar. If something is actually wrong, the error should appear in the red bar. If nothing is actually wrong, there should be no error bar/message.

### How to reproduce

1. spun up a localhost instance using astrocloud cli
2. Had two DAGs running - one DAG where I was using a BashOperator to use the `airflow backfill dags` command to backfill another DAG as a test. 

If relevant, the backfill trigger dag code is: 

```
from airflow import DAG
from airflow.operators.bash import BashOperator
from datetime import datetime

with DAG(dag_id='backfill_trigger_dag',
         schedule_interval=None,
         start_date=datetime(2022, 1, 1),
         catchup=False) as dag:

    trigger_backfill = BashOperator(
        task_id='trigger_backfill',
        bash_command='airflow dags backfill -s 20220401 -e 20220410 example_dag_basic',
    )

    trigger_backfill
```

and the DAG it was triggering is: 

```
from airflow import DAG
from airflow.operators.dummy import DummyOperator
from datetime import datetime, timedelta
from airflow.operators.python import PythonOperator
from airflow.operators.bash import BashOperator

default_args = {
    'retries': 0,
    'retry_delay': timedelta(minutes=5)
}


def pass_xcom(**kwargs):
    my_xcom = "This string will be passed as an xcom by returning it"
    return my_xcom


def pull_xcom(ti):
    my_xcom = ti.xcom_pull(key='return_value', task_ids=['pass_xcom'])
    print(my_xcom)


with DAG(dag_id='example_dag_basic',
         default_args=default_args,
         schedule_interval="@daily",
         start_date=datetime(2022, 1, 1),
         catchup=False) as dag:

    start = DummyOperator(
        task_id='start')

    pass_xcom_task = PythonOperator(
        task_id='pass_xcom_task',
        python_callable=pass_xcom
    )

    pull_xcom_task = PythonOperator(
        task_id='pull_xcom_task',
        python_callable=pull_xcom
    )

    bash_task = BashOperator(
        task_id='bash_task',
        bash_command='echo Hello, world!'
    )

    finish = DummyOperator(
        task_id='finish')

    start >> pass_xcom_task >> pull_xcom_task >> bash_task >> finish
```

### Operating System

macOS 11.5.2

### Versions of Apache Airflow Providers

apache-airflow-providers-amazon==3.2.0
apache-airflow-providers-celery==2.1.3
apache-airflow-providers-cncf-kubernetes==3.0.0
apache-airflow-providers-databricks==2.5.0
apache-airflow-providers-elasticsearch==3.0.2
apache-airflow-providers-ftp==2.1.2
apache-airflow-providers-google==6.7.0
apache-airflow-providers-http==2.1.2
apache-airflow-providers-imap==2.2.3
apache-airflow-providers-microsoft-azure==3.7.2
apache-airflow-providers-postgres==4.1.0
apache-airflow-providers-redis==2.0.4
apache-airflow-providers-slack==4.2.3
apache-airflow-providers-snowflake==2.6.0
apache-airflow-providers-sqlite==2.1.3

### Deployment

Astronomer

### Deployment details

AstroCloud CLI Version: 1.3.3
Docker Desktop 4.3.0

### Anything else

Today was the first time I have noticed this issue

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by bbovenzi** [comment]
I'll work on replicating this locally and figure out which steps to address:
- Prevent the UI from crashing on an error
- Prevent the error in the first place
- At least include a message in the error

**Comment by skyboi1233** [comment]
Why close it


Sent from Yahoo Mail for iPhone


On Monday, May 16, 2022, 5:43 PM, Brent Bovenzi ***@***.***> wrote:




Closed #23017.
